app/components/quiz/Quiz.js

```diff
@@ -1,70 +1,66 @@
-import React from 'react';
+import { newQuestions } from 'actions/actions';
+import classNames from 'classnames';
 import QuizQuestion from 'components/quiz/QuizQuestion.js';
-import 'Quiz.scss';
+import React from 'react';
 import ReactCSSTransitionsGroup from 'react-addons-css-transition-group';
+import { connect } from 'react-redux';
+
+import 'Quiz.scss';
 import 'fadeIn.scss';
-import classNames from 'classnames';
 
-export default React.createClass({
+const tmpquestions = [
```

Comment:
So are the questions and answers still not in the store? Or is this a temporary fix, and line 44 onward addresses that?